tests/DpAccordion.spec.js

```diff
@@ -0,0 +1,58 @@
+import { shallowMount } from '@vue/test-utils'
+import DpAccordion from '~/components/DpAccordion/DpAccordion.vue'
+import DpTransitionExpand from '~/components/DpTransitionExpand'
+
+describe('DpAccordion', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallowMount(DpAccordion, {
+      props: {
+        dataCy: 'accordionToggle',
+        fontWeight: 'bold',
+        compressed: false,
+        isOpen: false,
+        title: 'Test Title'
+      }
+    })
+  })
+
+  it('renders component correctly', () => {
```

Comment:
I find this more accurate than 'it renders correctly'

```suggestion
  it('the component is mounted', () => {
```